packages/cactus-plugin-htlc-eth-besu/src/test/solidity/contracts/DemoHelpers.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.7.3;
```

Comment:
@elenaizaguirre Same question as above

Reply:
@petermetz Same answer, I am testing openapi in this package because I need openapi.json document.